gtfs/spec/en/reference.md

```diff
@@ -378,9 +378,9 @@ For a given ordered pair of arriving trip and departing trip, the transfer with
 |  `to_stop_id` | Foreign ID referencing `stops.stop_id` | **Required** | Identifies a stop or station where a connection between routes ends. If this field refers to a station, the transfer rule applies to all child stops. |
```

Comment:
Should this be required to be of type stop, and not a station, when the transfer type is 4 or 5?